File: HackerRank/Python/Easy/E0078.py
# Problem Statement: https://www.hackerrank.com/challenges/np-sum-and-prod/problem

import numpy as np

N, M = np.array(input().split(), dtype='int32')
arr = np.array([input().split() for _ in range(N)], dtype='int32')
print(np.prod(np.sum(arr, axis=0)))

File: HackerRank/Algorithms/Easy/E0078.cpp
/*
Problem Statement: https://www.hackerrank.com/challenges/sherlock-and-the-beast/problem
*/

#include <iostream>
#include <string>

using namespace std;

string decentNumber(int n) {
	if (n % 3 == 0)
		return string(n, '5');
	int m = 5;
	while (m <= n) {
		if ((n - m) % 3 == 0)
			return string(n - m, '5') + string(m, '3');
		m += 5;
	}
	return "-1";
}

int main() {
	int t;
	cin >> t;
	while (t--) {
		int n;
		cin >> n;
		cout << decentNumber(n) << endl;
	}
	return 0;
}